Prevent duplicate template lists

johnhpercival opened the pull request:
Prevent duplicate template lists when initializer is called twice when there are two atto editors on a single page, as in #27

johnhpercival committed:
Prevent duplicate template lists

Prevent duplicate template lists when initializer is called twice when there are two atto editors on a single page.

johnhpercival committed:
Prevent duplicate template lists

Prevent duplicate template lists when initializer is called twice when there are two atto editors on a single page.

johnhpercival committed:
Prevent duplicate template lists

Prevent duplicate template lists when initializer is called twice when there are two atto editors on a single page.

johnhpercival committed:
Prevent duplicate template lists

Prevent duplicate template lists when initializer is called twice when there are two atto editors on a single page.

ralf-krause commented:
Thanks for this idea. I found that your code repository does not contain the changes from https://github.com/academic-moodle-cooperation/moodle-atto_bsgrid/pull/25

johnhpercival committed:
Apply PR/25 from master

Apply Pull Request #25 from original repository
https://github.com/academic-moodle-cooperation/moodle-atto_bsgrid/pull/25

johnhpercival commented:
I think I've been able to merge that into my repository

ralf-krause commented:
Thank you for doing this.
Please look at the comments on https://moodle.org/plugins/atto_bsgrid
